tests/conference/test_conference_service.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+
+def test_create_conference(session):
+    from dispatch.conference.service import create
+    from dispatch.conference.models import ConferenceCreate
+
+    name = "fakename"
+    resource_id = "000000"
+    resource_type = "resourcetype"
+    weblink = "https://www.example.com"
+    conference_id = "12345"
+    conference_challenge = "a0v0a0v9a"
+
+    conference_in = ConferenceCreate(
+        name=name,
```

Comment:
Remove. `name` does not exist in the conference model.

Reply:
You're right, thanks :taco:

Reply:
Removed in 08fe5c5.